libs/@guardian/libs/src/consent-management-platform/usnat/getConsentState.ts

```diff
@@ -1,18 +1,29 @@
 import { type USNATConsentState } from '../types/usnat';
 import { getGPPData } from './api';
 
+const getSupportedAPIv1 = (
+	supportedAPI: string | undefined,
+): string | undefined => {
+	if (supportedAPI?.includes('usnat')) {
+		return supportedAPI.includes('v1') ? supportedAPI : `${supportedAPI}'v1'`;
```

Comment:
I think you've got some extra quotes here?
```suggestion
		return supportedAPI.includes('v1') ? supportedAPI : `${supportedAPI}v1`;
```

Reply:
Thanks!